simple_history/tests/tests/test_models.py

```diff
@@ -1478,3 +1483,46 @@ def test_revert(self):
 
         self.one = Street.objects.get(pk=id)
         self.assertEqual(self.one.history.count(), 4)
+
+
+@override_settings(**database_router_override_settings_history_in_diff_db)
+class SaveHistoryInSeparateDatabaseTestCase(TestCase):
+    multi_db = True
+
+    def setUp(self):
+        self.model = ModelWithHistoryInDifferentDb.objects.create(name="test")
+
+    def test_history_model_saved_in_separate_db(self):
+        self.assertEqual(0, self.model.history.using("default").count())
```

Comment:
Interesting - the history table shouldn't exist at all the default DB, right? This kind of makes it seem like it does, but that it will remain empty.

Reply:
So the history table exists in both here. Testing the router's ability to migrate only to one table is a lot harder (since migrations happen before any tests are run). So I figured it was enough to test that the history table is being both written and read from the `"other"` database. @kseever

Reply:
hmm, I'm not sure I know enough about Django's ORM to tell if this is expected or not. Were you expecting an exception to be raised?

Reply:
@dopatraman had the router been present at the beginning of the tests (specified in the test settings as opposed to overriding the settings for these specific tests), then an error would be expected. But I think this is adequate testing without testing the migration.

Reply:
Agreed, I think these tests cover our expected use. @rossmechanic if you're ready, let's merge!